jenkins/bootstrap_test.py

```diff
@@ -2063,6 +2063,7 @@ def test_all_project_are_unique(self):
             'ci-kubernetes-kubemark-5-gce.env': 'ci-kubernetes-kubemark-*',
             'ci-kubernetes-kubemark-high-density-100-gce.env': 'ci-kubernetes-kubemark-*',
             'ci-kubernetes-kubemark-gce-scale.env': 'ci-kubernetes-scale-*',
+            'ci-kubernetes-e2e-gce-5000-performance.env': 'ci-kubernetes-scale-*',
```

Comment:
Can we not include "5000" in the name? I don't think it's a good thing to do.
I would rather go the way we went with kubemark and put something like "scale", "large" or anything like that in the name.

@gmarek